ckan/lib/datapreview.py

```diff
@@ -264,7 +264,9 @@ def add_views_to_resource(context,
             view = {'resource_id': resource_dict['id'],
                     'view_type': view_info['name'],
                     'title': view_info.get('default_title', _('View')),
-                    'description': view_info.get('default_description', '')}
+                    'title_fr': view_info.get('default_title', _('View')),
+                    'description': view_info.get('default_description', ''),
+                    'description_fr': view_info.get('default_description', '')}
```

Comment:
similar issue here, if a default description is provided it will be likely wrong in one language

Reply:
@wardi same questions as above.

But with this one, none of the core view plugins have `default_description` keys in their info dicts. And from what I can tell, it is the same on ckan master.

So I do not think we have to worry about the `description` and `description_fr` as we are not setting any `default_description` for any of the view plugin infos.

For now, I could put in a FIXME or TODO in regard to it. Just encase we do end up using `default_description` in the future.